.github/workflows/build.yml

```diff
@@ -42,5 +55,32 @@ jobs:
           path: download
       - name: create release
         run: |
-          release="$(.github/workflows/release.sh ${{ secrets.GITHUB_TOKEN }} ${{ github.repository }} create latest Builder)"
+          release="$(.github/workflows/release.sh ${{ secrets.GITHUB_TOKEN }} ${{ github.repository }} create latest "Builder (latest)")"
+          .github/workflows/release.sh ${{ secrets.GITHUB_TOKEN }} ${{ github.repository }} upload "$release" download/build
+
+  # Run for new intentional versions, bumping the major or minor version
+  release-semver:
```

Comment:
`semver` versioning would require `Major.Minor.Patch`. 
I would avoid using the name `semver` here, or we could simply enhance it to make it semver - https://semver.org

For the builder versions we are quite free to decide about the versioning. If we use semver, we should use it (as it is intended) to communicate what changed. 

(sorry for the nitpick here)

Reply:
Words have meanings, you're right, I chose a lose interpretation of 'semantic versioning' here.. will rename